libnmstate/state.py

```diff
@@ -132,6 +188,20 @@ def verify_interfaces(self, other_state):
 
         self._assert_interfaces_equal(other_state)
 
+    def verify_routes(self, other_state):
+        self._clean_routes()
+        other_state._clean_routes()
+        for iface_name, routes in six.viewitems(self.iface_routes):
```

Comment:
Why are you comparing by interface? Couldn't you use `set.issubset()` instead after converting the routes in sets?

Reply:
The performance of `subset` is worse than equal check.

Reply:
I do not understand why the comparison is per interface, regardless of how it is compared (subset or equality).
How is this any different from just doing:
`self.state['routes']['config'] != other_state.state['routes']['config']`
(assuming these are sorted like we do with the interfaces)